MykhailoTymoshchuk/HW11/task2.py

```diff
@@ -0,0 +1,31 @@
+week_days = {1: "Mondey" , 2: "Thuesday", 3: "Wednesday", 4: "Thursday", 
+             5: "Friday", 6: "Saturday", 7: "Sunday"}
+
+
```

Comment:
look at this version of code:
d = {
    1: 'Monday',
    2: 'Tuesday',
    3: 'Wednesday',
    4: 'Thursday',
    5: 'Friday',
    6: 'Saturday',
    7: 'Sunday'
}

while True:
    try:
        i = int(input('Enter the day of the week: '))
    except ValueError:
        print('You did not enter a number!')
    else:
        print(d.get(i, 'There is no such day of the week!'))